_usage/limitations-of-babelfish.md

```diff
@@ -9,89 +9,90 @@ nav_order: 5
 In this chapter, you will learn about Babelfish limitations and compatibility.
```

Comment:
I don't think this chapter actually discusses 'compatibility'. We should anyway avoid that term as much as possible.

Reply:
I beg your pardon?
This is all about compatibility.
In fact, all of Babelfish is.

Reply:
From an amazon perspective, we avoid this term. Customers may call it like that, but we should not. THis has to do with reason I am not going to put in writing.

Reply:
Ok, I defer to you on that. Removed.